crates/trycmd/src/schema.rs

```diff
@@ -645,12 +645,27 @@ pub struct OneShot {
     #[serde(default)]
     pub(crate) binary: bool,
     #[serde(default)]
-    #[serde(deserialize_with = "humantime_serde::deserialize")]
+    #[serde(deserialize_with = "deserialize_jiff_duration")]
     pub(crate) timeout: Option<std::time::Duration>,
     #[serde(default)]
     pub(crate) fs: Filesystem,
 }
 
+fn deserialize_jiff_duration<'de, D>(
+    deserializer: D,
+) -> Result<Option<std::time::Duration>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    use serde::de::Deserialize;
+
+    let buf = String::deserialize(deserializer)?;
+
+    buf.parse::<jiff::SignedDuration>()
+        .map(|e| std::time::Duration::try_from(e).ok())
```

Comment:
If the value cannot be represented, we should error, rather than act as if there is no timeout

Reply:
Should it return nicely or should we `unwrap`. I guess `unwrap` from your previous comment

Reply:
My previous comment was that we should *not* be doing unwraps for these.

Reply:
Ok sorry for the confusion. Updating this last bit soon. Thanks for the repeated review

Reply:
Updated